### PR TITLE
Bug 1106562 — PATCH /rooms documentation

### PR DESCRIPTION
**Pull request by Natim**
https://bugzilla.mozilla.org/show_bug.cgi?id=1106562

### CODE REVIEW

File: source/loop/apis.rst [hunk 1]
```diff
@@ -914,6 +914,57 @@ POST /rooms
...
+
+    Remove given rooms
+
+    Request body parameters:
+
+    - **deleteRoomTokens**, a list of rooms to delete.
+
+    Response body parameters:
+
+    - **deletedRoomTokens**, a list of room's tokens actually deleted.
```
Comment (ametaireau) [inline source/loop/apis.rst]: This doesn't match what is implemented. The current implementation return `responses`.  see https://github.com/mozilla-services/loop-server/pull/272/files#diff-2fbbe66e501e7e82cd4c1868a4632212R525

Comment (ametaireau) [inline source/loop/apis.rst]: I propose we keep this response and update the actual implementation.

File: source/loop/apis.rst [hunk 2]
```diff
@@ -914,6 +914,57 @@ POST /rooms
...
+
+    Potential HTTP error responses include:
+
+    - **404 Not Found:**  If none of the given roomTokens where found for this user.
+
+    Example::
+
+        echo '{"deleteRoomTokens": ["pPVoaqiH89M"]}' | http PATCH localhost:5000/v1/rooms -v \
```
Comment (ametaireau) [inline source/loop/apis.rst]: Can we pass the arguments to httpie directly? (rather than doing the echo trick).

File: source/loop/apis.rst [hunk 3]
```diff
@@ -1343,7 +1398,7 @@ Participant information
       session.
     - **owner**, if the user is also the owner of the room, this
       property will be true, it will be false otherwise.
-    - **id**, An id, unique within the room for the
+    - **roomConnectionId**, An id, unique within the room for the
```
Comment (ametaireau) [inline source/loop/apis.rst]: thanks.

File: source/loop/apis.rst (hunk 2 above)
Comment (Natim) [inline source/loop/apis.rst]: Not that I know of:

    $ http -v POST http://localhost/test toto='["titi", "tutu"]'
    {
      "toto": "[\"titi\", \"tutu\"]"
    }

    $ http -v POST http://localhost/test toto=titi,tutu
    {
        "toto": "titi,tutu"
    }

**Pull request merged**